cpp/src/io/comp/debrotli.cu

```diff
@@ -154,25 +156,34 @@ struct debrotli_state_s {
   debrotli_huff_tree_group_s *distance_hgroup;
   uint16_t *block_type_vlc[3];
   huff_scratch_s hs;
-  uint32_t mtf[65];
-  uint64_t heap[LOCAL_HEAP_SIZE / 8];
+  uint8_t mtf[65 * sizeof(uint32_t)];
+  char heap[LOCAL_HEAP_SIZE];
 };
 
 inline __device__ uint32_t Log2Floor(uint32_t value) { return 32 - __clz(value); }
 
 /// @brief initializes the bit reader
 __device__ void initbits(debrotli_state_s *s, const uint8_t *base, size_t len, size_t pos = 0)
 {
-  const uint8_t *p      = base + pos;
-  uint32_t prefix_bytes = (uint32_t)(((size_t)p) & 3);
+  const uint8_t *p = base + pos;
+
+  auto const prefix_bytes = reinterpret_cast<size_t>(p) & 3;
```

Comment:
another `uint32_t` -> `uint64_t` change. just want to make sure they're intentional.